src/main/java/io/github/railroad/vcs/RepositoryTypes.java

```diff
@@ -0,0 +1,5 @@
+package io.github.railroad.vcs;
+
+public enum RepositoryTypes {
+    git, svn
```

Comment:
These should be `GIT` and `SVN`. You'll have to rename those in the IDE, I can't do it here because it'd break usages.